### PR TITLE
Update README badges

### PR DESCRIPTION
**Pull request by romanlutz**
- Reducing clutter by removing npm packages that aren't user-facing
- adding labels to PyPI badges
- adding responsibleai package badge
- putting badges of different kinds on separate lines
- 
Result:
image

## Checklist

- [ ] I have added screenshots above for all UI changes.
- [ ] I have added e2e tests for all UI changes.
- [ ] Documentation was updated if it was needed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,10 @@
...
+![MIT license](https://img.shields.io/badge/License-MIT-blue.svg)
+
+![Responsible AI Widgets Python Build](https://github.com/microsoft/responsible-ai-widgets/workflows/Responsible%20AI%20Widgets/badge.svg) ![CD](https://github.com/microsoft/responsible-ai-widgets/workflows/CD/badge.svg)
+
+![PyPI raiwidgets](https://img.shields.io/pypi/v/raiwidgets?label=PyPI%20raiwidgets) ![PyPI responsibleai](https://img.shields.io/pypi/v/responsibleai?label=PyPI%20responsibleai)
```
Comment (gaugup) [inline README.md:5]: Do we want to include all pypi packages shipping out of this repo? erroranalysis, raiutils, rai_test_utils seems to be missing.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,10 @@
...
+![MIT license](https://img.shields.io/badge/License-MIT-blue.svg)
+
+![Responsible AI Widgets Python Build](https://github.com/microsoft/responsible-ai-widgets/workflows/Responsible%20AI%20Widgets/badge.svg) ![CD](https://github.com/microsoft/responsible-ai-widgets/workflows/CD/badge.svg)
```
Comment (gaugup) [inline README.md:3]: What is CD? Could we rename this?

File: README.md (hunk 1 above)
Comment (romanlutz) [reply README.md:5]: Are we actually expecting anyone to use them? I included the ones that are actually used by people.

File: README.md (hunk 2 above)
Comment (gaugup) [reply README.md:3]: "CD = continuous deployment" doesn't communicate what it is. Could we rename this here something else more meaningful?

File: README.md (hunk 1 above)
Comment (romanlutz) [reply README.md]: Done

File: README.md [hunk 3]
```diff
@@ -1,4 +1,13 @@
...
```
Comment (romanlutz) [inline README.md:3]:
```suggestion
![Responsible AI Widgets Python Build](https://img.shields.io/github/actions/workflow/status/microsoft/responsible-ai-toolbox/CI-raiwidgets-pytest.yml?branch=main&label=Responsible%20AI%20Widgets%20Python%20Build)
![UI deployment to test environment](https://img.shields.io/github/actions/workflow/status/microsoft/responsible-ai-toolbox/CD.yml?branch=main&label=UI%20deployment%20to%20test%20environment)
```

File: README.md (hunk 2 above)
Comment (romanlutz) [reply README.md]: Updated and fixed the link which turned out to be broken.

**Review by gaugup — APPROVED** [review]

**Review by imatiach-msft — APPROVED** [review]

**Pull request merged**